src/components/PermissionsManagement/PermissionsManagement.jsx

```diff
@@ -122,7 +122,7 @@ function PermissionsManagement(props) {
                   Add New Role
                 </Button>
               )}
-              {canManageUserPermissions && (
+              {canManageUserPermissions && canPutRole && (
```

Comment:
```suggestion
              {canManageUserPermissions && (
```
This conditional render is just for the manage user permissions button. `canPutRole` controls the conditional render of the role list (where you could select an existing role to edit).